backend/cmd/headlamp.go

```diff
@@ -1108,6 +1108,13 @@ func (c *HeadlampConfig) getClusters() []Cluster {
 			continue
 		}
 
+		// this function handles the clusters that need to have headlamp_info created or updated
+		err := c.handleHeadlampInfo(context)
+		if err != nil {
+			logger.Log(logger.LevelError, map[string]string{"cluster": context.Name},
+				err, "failed to handle headlamp_info")
+		}
```

Comment:
Since level of error is already `LevelError` we do not need to be rendundant with the log message. Can you please update it everywhere? Thanks
```suggestion
		if err := c.handleHeadlampInfo(context); err != nil {
			logger.Log(logger.LevelError, map[string]string{"cluster": context.Name},
				err, "handling headlamp_info")
		}
```

Reply:
fixed the string!